src/Phpbb/Area51Bundle/Resources/views/Default/index.html.twig

```diff
@@ -3,34 +3,34 @@
 {% block body %}
 <div class="clearfix">
     <h2>Get Involved</h2>
-    <p>You can help the project in many ways. Pick what suits your skills best, all contributions are much appreciated by the rest of the community. More detailed information on how to go about contributing is available on our <a href="http://wiki.phpbb.com/Get_Involved" title="Get Involved">Get Involved</a> wiki page.</p>
```

Comment:
Hasn't this stuff not moved over to the dev docs?

Reply:
Some of it, although hanakin worked on moving the coding guidelines etc. Will update that soonish afterwards.